### PR TITLE
Update e2e environment

### LINKED ISSUE #2223: Update the integration image
## Description
Update the in integration image
- [x] Geth node
- [x] Pathfinding service
- [x] Synapse

## Acceptance criteria
- 

## Tasks
- [ ]

### PR DESCRIPTION
**Pull request by palango**
**Thank you for submitting this pull request :)**

Fixes #2223 

**Short description**

Update the e2e environment to the latest version of dependencies.

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

**Comment by palango** [comment]
@andrevmatos With those changes, when I run the 2e2 test locally I run into this problem. Is this a problem with the persistence layer?
```
  console.info
     action     { type: 'raiden/shutdown', payload: { reason: 'raidenStopped' } }

      at ../node_modules/.pnpm/redux-logger@3.0.6/node_modules/redux-logger/dist/redux-logger.js:1:6388
          at Array.forEach (<anonymous>)

  console.warn
    Persister saveDatabase error TypeError: Right-hand side of 'instanceof' is not callable
        at new ReadableState (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/sublevel-pouchdb/node_modules/readable-stream/lib/_stream_readable.js:105:14)
        at ReadStream.Readable (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/sublevel-pouchdb/node_modules/readable-stream/lib/_stream_readable.js:139:25)
        at new ReadStream (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/sublevel-pouchdb/lib/index.js:289:12)
        at ReadStream (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/sublevel-pouchdb/lib/index.js:286:12)
        at EventEmitter.emitter.readStream.emitter.createReadStream (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/sublevel-pouchdb/lib/index.js:261:14)
        at /Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/pouchdb-adapter-leveldb-core/lib/index.js:951:41
        at countDocs (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/pouchdb-adapter-leveldb-core/lib/index.js:289:12)
        at /Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/pouchdb-adapter-leveldb-core/lib/index.js:906:7
        at tryCode (/Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/pouchdb-adapter-leveldb-core/lib/index.js:315:11)
        at /Users/paul/Projects/brainbot/light-client/node_modules/.pnpm/pouchdb-adapter-memory@7.2.2/node_modules/pouchdb-adapter-memory/node_modules/pouchdb-adapter-leveldb-core/lib/index.js:362:7



      at src/persister.ts:2166:49
```

**Comment by palango** [comment]
Tested this locally and it works.
@weilbith How do I update this at docker hub?

**Comment by weilbith** [comment]
At the moment just label it with `latest` and push it. There is only an open issue to improve that procedure. Until then that's how it is.

File: e2e-environment/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ARG RAIDEN_VERSION="v1.1.1"
 ARG CONTRACTS_PACKAGE_VERSION="0.37.1"
 ARG CONTRACTS_VERSION="0.37.0"
-ARG SERVICES_VERSION="645e106d406b6fc3e11441c8e85c76d3aff300d3"
-ARG SYNAPSE_VERSION="1.10.1"
+ARG SERVICES_VERSION="v0.13.1"
+ARG SYNAPSE_VERSION="v1.19.1"
```
Comment (weilbith) [inline e2e-environment/Dockerfile:5]: Why not using the latest version? :thinking: 
Or is this just the same as for the Python client?

File: e2e-environment/Dockerfile [hunk 2]
```diff
@@ -24,7 +24,7 @@ FROM python:3.7 as synapse-builder
 ARG SYNAPSE_VERSION
 
 RUN python -m venv /synapse-venv \
-    && /synapse-venv/bin/pip install "matrix-synapse[sqlite]==${SYNAPSE_VERSION}" \
+    && /synapse-venv/bin/pip install "matrix-synapse==${SYNAPSE_VERSION}" \
```
Comment (weilbith) [inline e2e-environment/Dockerfile:27]: Which database is Synapse using per default? What was again the issue with SQLite?

File: e2e-environment/synapse/auth/admin_user_auth_provider.py [hunk 3]
```diff
@@ -2,54 +2,54 @@
...
-
-from twisted.internet import defer
+from typing import Any
 
 
 class AdminUserAuthProvider:
     __version__ = "0.1"
 
-    def __init__(self, config, account_handler):
+    def __init__(self, config, account_handler) -> None:  # type: ignore
```
Comment (weilbith) [inline e2e-environment/synapse/auth/admin_user_auth_provider.py:11]: What is the purpose of adding the return type but then ignore type checks for this line? :thinking:

**Review by weilbith — APPROVED** [review]
Looks pretty good. :+1: 
I like the `async`/`await` syntax so much more...
The image got already re-build and uploaded? Or do you lack permission for the DockerHub repo?
Optimistic approval.

File: e2e-environment/Dockerfile (hunk 1 above)
Comment (palango) [reply e2e-environment/Dockerfile:5]: I went for the versions that run in the service bundle (and so on mainnet): https://github.com/raiden-network/raiden-service-bundle/blob/master/BUILD_VERSIONS

File: e2e-environment/Dockerfile (hunk 2 above)
Comment (palango) [reply e2e-environment/Dockerfile:27]: Seems like the default is sqlite. Here `pip`was complaining that `sqlite` was not a valid option name.

File: e2e-environment/synapse/auth/admin_user_auth_provider.py (hunk 3 above)
Comment (palango) [reply e2e-environment/synapse/auth/admin_user_auth_provider.py:11]: I also just copied this code from the RSB (raiden service bundle), I guess Ulo didn#t want to spend more time ...

Comment (weilbith) [reply e2e-environment/synapse/auth/admin_user_auth_provider.py:11]: Ah okay, I wasn't aware about that. So it's just fine.

PS: Another hint that these two environments should attempt to share some logic... :thinking:

**Pull request merged**